denguefever_tw/dengue_linebot/denguebot_fsm/denguebotfsm.py

```diff
@@ -213,6 +221,44 @@ def _create_hospitals_msgs(self, hospital_list):
         hospital_messages = [template_message]
         return hospital_messages
 
+    def _create_zapper_imgmap(self, event):
+        zapper_imgmap = ImagemapSendMessage(
+            base_url='https://i.imgur.com/9piGQjS.jpg',
+            alt_text='user zapper information',
+            base_size=BaseSize(height=1040, width=1040),
+            actions=[
+                URIImagemapAction(
```

Comment:
What is this empty block for?

Reply:
Some line user may not bind zapper, but i still send the imagemap to them.
But... i just tried the imagemap, and an error occurred.
The link_url can not be empty.

Reply:
I see

Reply:
I find a solution :smile:

Reply:
Great